src/client/src/ui/header/model/headerModel.js

```diff
@@ -1,76 +0,0 @@
-import { Router,  observeEvent } from 'esp-js/src';
-import { CompositeStatusService, FakeUserRepository } from '../../../services';
-import { logger } from '../../../system';
-import { ModelBase } from '../../common';
-import { ServiceStatusLookup } from '../../../services/model';
-
-var _log:logger.Logger = logger.create('HeaderModel');
-
-export default class HeaderModel extends ModelBase {
```

Comment:
I think rather than deleting this guy why not rename it to `FooterModel` and same with the associated view? Basically what we want to do here is just move the header to the footer.

This would be easier to rename now rather than bring back later.